image/scanner/scripts/import-additional-cas

```diff
@@ -1,8 +1,16 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Copied from https://github.com/stackrox/stackrox/blob/master/image/static-bin/import-additional-cas but removed `copy_existing /etc/pki/injected-ca-trust` because nobody injects CAs into Scanner